alpha/apps/kaltura/lib/myUploadUtils.class.php

```diff
@@ -250,5 +251,30 @@ public static function isFileTypeRestricted($fullPath, $partnerId = null)
 		return false;
 	}
 	
+	public static function handleRestrictedFilesByPath($fullPath)
+	{
+		if(myUploadUtils::isFileTypeRestricted($fullPath))
+		{
+			throw new KalturaAPIException(KalturaErrors::FILE_CONTENT_NOT_SECURE);
+		}
+	}
+	
+	public static function handleRestrictedFiles($fileData)
```

Comment:
why do we need two separate functions?
it will be better if we can move the fileSize check to the inner file type restriction function and just get the file path in the function